dashbuilder/dashbuilder-backend/dashbuilder-external-backend/src/test/java/org/dashbuilder/external/impl/ComponentAssetProviderImplTest.java

```diff
@@ -31,13 +31,13 @@
 import org.junit.runner.RunWith;
 import org.mockito.InjectMocks;
 import org.mockito.Mock;
-import org.mockito.runners.MockitoJUnitRunner;
+import org.mockito.junit.MockitoJUnitRunner;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
 import static org.mockito.Mockito.when;
 
-@RunWith(MockitoJUnitRunner.class)
+@RunWith(MockitoJUnitRunner.Silent.class)
```

Comment:
Some tests have many unnecessary mocks and it was easier for us to disable the checks (using this Silent class) than to actually remove the unnecessary mocks. Also, sometimes the mocks occur in a parent class, making it impossible for us to remove it, since some of their subclasses use those mocks, and some don't.